src/js/components/Image/StyledImage.js

```diff
@@ -18,6 +18,10 @@ const StyledImage = styled.img`
   ${genericStyles}
   ${props => props.fit && fitStyle}
   ${props => props.theme.image && props.theme.image.extend}
+  opacity: ${props =>
+    props.opacity === true
```

Comment:
`=== true` is probably not needed. just `props.opacity ? ..`

Reply:
hm this line actually seems weird 
`props.opacity === true? props.theme.global.opacity.medium  : props.theme.global.opacity[props.opacity] || props.opacity}`
why would we apply an opacity if user set opacity to false?
I think we should do something `${props => props.opacity && opacityStyle(props.opacity)}`

Reply:
what is the problem with opacity={false}? it works fine on the tests. 
not allowing `opacity={false}` cross grommet will be a regression to other components that use the opacity logic.

Reply:
well now you fixed it. before you were actually setting `opacity: false` as a css property ;)